ansible/roles/bastion/tasks/main.yml

```diff
@@ -34,8 +34,14 @@
     - copy_env_private_key
 
 - name: Generate host .ssh/config Template
-  become: no
-  local_action: template src={{ role_path }}/templates/bastion_ssh_config.j2 dest={{output_dir}}/ssh-config-{{ env_type }}-{{ guid }}
+  become: false
+  local_action: >-
+    template src={{
+    role_path
+    }}/templates/bastion_ssh_config.j2 dest={{
+    output_dir
+    }}/ssh-config-{{
+    env_type }}-{{ guid }}
```

Comment:
same here as for bastion-lite, no inlining

Reply:
Have made the change, but am unsure if it will do the same thing as the original.